### PR TITLE
[vrops-exporter] Cluster DRS alert

### PR DESCRIPTION
**Pull request by christopherhans**
Playbook PR is already raised, will add when raising this alert to warning.

### CODE REVIEW

File: prometheus-exporters/vrops-exporter/alerts/vccluster.alerts [hunk 1]
```diff
@@ -74,3 +74,17 @@ groups:
     annotations:
       description: "VC {{ $labels.vcenter }} {{ $labels.vccluster }} has NO failover host amount configured, this should be 1"
       summary: "VC {{ $labels.vcenter }} {{ $labels.vccluster }} has NO failover host amount configured, this should be 1"
+
+  - alert: ClusterDRSNotFullyAutomated
```
Comment (richardtief) [inline prometheus-exporters/vrops-exporter/alerts/vccluster.alerts:78]:
```suggestion
  - alert: VCClusterDRSNotFullyAutomated
```

**Review by richardtief — COMMENTED** [review]
Looks good. One suggestion to distinguish between all sorts of clusters.

File: prometheus-exporters/vrops-exporter/alerts/vccluster.alerts [hunk 2]
```diff
@@ -74,3 +74,17 @@ groups:
...
+      expr: vrops_cluster_configuration_drsconfig_defaultvmbehavior{state != "fullyAutomated"} == 0
```
Comment (richardtief) [inline prometheus-exporters/vrops-exporter/alerts/vccluster.alerts:79]: prom rule listing fails. Maybe remove the spaces. 

`79 prometheus-exporters/vrops-exporter/alerts/vccluster.alerts: yaml: line 79: mapping values are not allowed in this context`

**Review by richardtief — CHANGES_REQUESTED** [review]

File: prometheus-exporters/vrops-exporter/alerts/vccluster.alerts (hunk 2 above)
Comment (christopherhans) [reply prometheus-exporters/vrops-exporter/alerts/vccluster.alerts]: Don't know how that slipped through

**Review by richardtief — APPROVED** [review]
Thanks.

**Pull request merged**